test/common/quic/envoy_quic_server_session_test.cc

```diff
@@ -313,6 +313,7 @@ TEST_F(EnvoyQuicServerSessionTest, NewStream) {
   headers.OnHeader(":authority", host);
   headers.OnHeader(":method", "GET");
   headers.OnHeader(":path", "/");
+  headers.OnHeader(":scheme", "https");
```

Comment:
Hm. It looks like there is a behavior change in QUICHE around the :scheme header being required. Is that right? If so, we probably need to control this by an Envoy feature flag.

Reply:
Yes it's a result of https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quiche/quic/core/quic_flags_list.h;l=61?q=quic_flag&ss=chromium being flipped.
Maybe we should guard this flag with an Envoy flag?